packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -45,31 +44,54 @@ const makeDust = brand => AmountMath.make(brand, 1n);
  * @param {ZCFSeat} poolSeat
  * @param {ShareWorth} shareWorth
  * @param {Brand} USDC
+ * @param {Amount<'nat'>} outstandingLends
  */
-const checkPoolBalance = (poolSeat, shareWorth, USDC) => {
+const checkPoolBalance = (poolSeat, shareWorth, USDC, outstandingLends) => {
   const available = poolSeat.getAmountAllocated('USDC', USDC);
   const dust = makeDust(USDC);
-  isEqual(add(available, dust), shareWorth.numerator) ||
+  const virtualTotal = add(add(available, dust), outstandingLends);
+  isEqual(virtualTotal, shareWorth.numerator) ||
```

Comment:
```suggestion
  const encumberedBalance = add(add(available, dust), outstandingLends);
  isEqual(encumberedBalance, shareWorth.numerator) ||
```

Reply:
`checkPoolBalance` docstring needs updating

Reply:
Since encumbered and unencumbered balance are mutually exclusive, I don't think `encumberedBalance` is a good name here.

I did rename `outstandingLends` to `encumberedBalance` and use `unencumberedBalance` to represent the current poolSeat allocation. I'm calling the sum of encumbered and unencumbered `grossTotal`, but am open to additional suggestions.